source/reference/limitations.txt

```diff
@@ -32,16 +32,8 @@ General Limitations
   6.2. Only major MongoDB releases such as 6.0 or 7.0. are supported.
   For more information on MongoDB versioning, see
   :ref:`release-version-numbers`.
-- The source and destination clusters must have the same major and minor
-  release version, but can have different patch releases.
-
```

Comment:
Can we revert the updates made to versioning.txt made from this [PR](https://github.com/mongodb/docs-cluster-to-cluster-sync/pull/160/files#diff-6c476215b8a109eb3828e67d96bdb157de4bce0f09d7470973baddf92baa1f76L63) ?

Reply:
Fixed